src/plone/staticresources/_scripts/register_icons.py

```diff
@@ -0,0 +1,28 @@
+#! /usr/bin/env python
```

Comment:
Maybe you also want to add a ``plone-register-icons`` script entry point for that script here, so that you can call it via ``./bin/plone-register-icons``. Like: https://github.com/plone/plone.staticresources/blob/master/setup.py#L44

Reply:
I'm not sure if this is a good idea. The script uses relative paths and works only in this location. I'd leave it for now and add some improvements and docs how to use it e.g. in a collective.icons.xxx package.

Reply:
Okay, then wait until it's ready.